src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --maincolor: #00acdc;
+  --header-height: 60px;
+  --header-margin: 50px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .logo {
+    display: flex;
+    align-items: center;
+    margin-left: var(--header-margin);
+  }
+
+  .nav {
+    margin: 0 var(--header-margin) 0 0;
+  }
+
+  .nav ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+
+    :not(:last-child) {
+      margin-right: 20px;
+    }
+
+    .nav__item {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .nav__item a.is-active::after {
+      content: '';
+      position: absolute;
+      width: 100%;
+      height: 4px;
+      background-color: var(--maincolor);
+      border-radius: 8px;
+      bottom: 0;
+    }
```

Comment:
CSS rules for `.nav__item a.is-active::after` should not be nested inside `.nav ul`. Correct the nesting to ensure proper CSS syntax.